amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3904,98 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventsInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventInfo'
+  EventInfo:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+  EventBodyCreate:
+    type: object
+    required:
+      - title
+    properties:
+      name:
+        type: string
+      type:
+        type: string
+        default: Simple
+        enum:
+          - Simple
+          - Obsoleted
+          - Superseded
+          - Cross-Referenced Record Transferred
+          - Versioned
+  EventEntry:
+    type: object
+    required:
+      - name
+      - type
+    properties:
+      name:
+        type: string
+      type:
+        type: string
+  EventBodyUpdate:
+    type: object
+    required:
+      - id
+    properties:
+      name:
+        type: string
+      type:
+        type: string
+        default: Simple
+        enum:
+          - Simple
+          - Obsoleted
+          - Superseded
+          - Cross-Referenced Record Transferred
+          - Versioned
+  EventTypeInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventTypeEntry'
+  EventTypeEntry:
+    type: object
+    required:
+      - id
+      - name
+      - isAutomatic
+      - associationName
```

Comment:
`associationName` is not mandatory. I would also consider making `isAutomatic` have a default value of `true`.

Reply:
Done